easybuild/easyblocks/s/superlu.py

```diff
@@ -93,6 +93,12 @@ def configure_step(self):
             openblas_lib = os.path.join(get_software_root('OpenBLAS'), get_software_libdir('OpenBLAS'), "libopenblas.a")
             self.cfg.update('configopts', '-DBLAS_LIBRARIES="%s;-pthread"' % openblas_lib)
 
+        elif toolchain_blas == 'FlexiBLAS':
+            # Unfortunately, FlexiBLAS is not recognized by FindBLAS from CMake,
```

Comment:
Teaches me a lesson to do only copy&pasta! :D 
That said: without the Python changes it does not find `FlexiBlas`. 
See also this [PR](https://github.com/easybuilders/easybuild-easyconfigs/pull/15272)

Reply:
You can add a CMake version check and just use
```
self.cfg.update('configopts', '-DBLA_VENDOR="FlexiBLAS"')
```
and the same for OpenBLAS (which is supported since CMake 3.6)

Reply:
Done some further digging. It turned out the `-Denable_blaslib=OFF` has been replaced by `-Denable_internal_blaslib=OFF` and thus the internal `BLAS` was used. Fixed that now. 
Also tidied up the check for `OpenBLAS` and `FlexiBLAS`. Please check if that is ok the way I done it in light of consistency with older installations. 
Thanks for your help!